.github/workflows/elixir.yml

```diff
@@ -32,20 +31,65 @@ jobs:
       uses: actions/cache@v2
       with:
         path: deps
-        key: ${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-${{ hashFiles('**/mix.lock') }}
-        restore-keys: ${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-
+        key: test-${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-${{ hashFiles('**/mix.lock') }}
+        restore-keys: test-${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-mix-
 
     - name: Cache build
       uses: actions/cache@v1
       with:
         path: _build
-        key: ${{ runner.OS }}-${{ matrix.elixir }}-${{ matrix.otp }}-build-
+        key: test-${{ runner.OS }}-${{ matrix.elixir }}-${{ matrix.otp }}-build-
         restore-keys: |
-          ${{ runner.OS }}-${{ matrix.elixir }}-${{ matrix.otp }}-build-
+          test-${{ runner.OS }}-${{ matrix.elixir }}-${{ matrix.otp }}-build-
 
     - name: Install dependencies
       run: |
         mix deps.get
 
     - name: Run tests
       run: mix test --timeout 120000
+
+  lint:
```

Comment:
Took the liberty of running these in parallel, not sure if it's something we actually don't want.

Reply:
To be honest, I don't think we want it. mix formatter is extremely fast. so by paralellizing it you're probably just wasting time and resources setting up a new instance

Reply:
Technically, it's not just `mix formatter`, there's also credo. But yeah. I think usually like these split since I like to have easy access to info if it's tests or lints failing. It's not a necessity. I can just collapse everything to a single job